modules/services/conky.nix

```diff
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.conky;
+
+  configFile = if cfg.settings != "" then
+    cfg.settings
+  else
+    builtins.readFile (pkgs.runCommand "conky.conf" { }
+      "${pkgs.conky}/bin/conky --print-config > $out");
```

Comment:
@ambroisie I'm not sure if it is a good idea to get the default config  from command output.  maybe  we just hardcode the default config in here?

Reply:
Sorry, that's not what I meant when I said one can use `readFile`: you'd introduced an option to give a path to a config file, instead of using the `settings` attribute. Instead one could just do `settings = readFile`.

Definitely a :-1: on doing this, as it's an IFD.

Reply:
ok, I misunderstood the comments before.   

> Definitely a 👎 on doing this, as it's an IFD.

let me try to  summary  the config/settings part:
- keep  `settings` option only ( as `types.lines` )
- no default config, it means
   - if `settings` is not `""`, then we generate the xdg confile file and append `--config {filepath}` to `ExecStart`
   - else only run `conky`  in  `ExecStart`. in that case Conky will then use the builtin-configs to start. 

so we don't have to run the external command to generate or hardcode a config in here. does it make sense to you?